webchat/components/toggle-button/toggle-button.html

```diff
@@ -0,0 +1,15 @@
+<md-button
+        class="md-icon-button"
+        ng-switch="toggleButtonCtrl.active"
+        ng-click="toggleButtonCtrl.toggle()" >
+    <md-icon
+            ng-switch-when="true"
```

Comment:
This could probably be done with [ngClass](https://docs.angularjs.org/api/ng/directive/ngClass)...